packages/webapp/src/hooks/useScrollBox.ts

```diff
@@ -58,25 +71,50 @@ function useScrollBox(scrollRef: any) {
     smoothScroll(target)
   }
 
+  // Throttled function to update the last horizontal scroll position, minimizing performance impact.
   const handleLastScrollX = useCallback(
     throttle((screenX) => {
       setLastScrollX(screenX)
     }, timing),
     []
   )
 
+  // Handles the wheel event to adjust the scrollLeft property, enabling horizontal scrolling.
   const handleScroll = (e: WheelEvent) => {
     if (scrollRef.current) {
       // Adjust the scrollLeft property based on the deltaY value
       scrollRef.current.scrollLeft += e.deltaY
     }
   }
 
+  // Responds to window resize events to update arrow visibility states.
   const handleResize = () => {
     setShowLeftArrow(true)
     setShowRightArrow(true)
   }
 
+  // Smoothly scrolls to the rightmost end of the container and then back to the start.
+  const smoothScrollToRightThenLeft = useCallback(() => {
+    const element = scrollRef.current
+    if (!element) return
+
+    const targetRight = element.scrollWidth - element.clientWidth
+    smoothScroll(targetRight, () => {
+      triggerLastCardGlow()
+      setTimeout(() => {
+        smoothScroll(0) // Scrolls back to the first element
```

Comment:
Wait, why?

Reply:
I just thought it was a nice touch from an experience pov, i'll remove this :)